### PR TITLE
MCKIN-22514 rounding off correctly

### PR DESCRIPTION
**Pull request by mudassir-hafeez**
Jira ticket https://edx-wiki.atlassian.net/browse/MCKIN-22514

this PR keeps up to date master for #297 
> There is an endpoint that was missed in #267 which needs to be addressed.

### CODE REVIEW

**Comment by xitij2000** [comment]
@mudassir-hafeez I think solutions still uses version 3 so you will also need a PR against the v3 branch.

File: setup.py [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 # Constants #########################################################
 
-VERSION = '4.0.6'
+VERSION = '3.4.23'
```
Comment (aayushagra) [inline setup.py:31]: @mudassir-hafeez Why is this downgraded?

**Review by aayushagra — COMMENTED** [review]
Hi @mudassir-hafeez can you please explain the following change:

File: setup.py (hunk 1 above)
Comment (mudassir-hafeez) [reply setup.py:31]: @aayushagra thank you for taking my attention towards it. Actually, if you see releases, we were using two different versions of releases. one is for Ironwood Open edX releases and other is for normal releases. After Ironwood upgrade, releases should be as they were before. @xitij2000 can you please comment on that what should be the right release version from now.

**Comment by mudassir-hafeez** [comment]
Oh got it, we are using 4 series of releases for master and 3 series against the v3 branch

**Comment by mudassir-hafeez** [comment]
@aayushagra I have updated the version accordingly. can you please review it once more? thanks

**Comment by aayushagra** [comment]
@mudassir-hafeez Looks good. Can you please squash your changes down to one commit before merging?

**Comment by aayushagra** [comment]
:+1:

- [x] I tested this: Tested that the code works as intended and tests pass
- [x] I read through the code
- [] I checked for accessibility issues
- [] Includes documentation
- [] I made sure any change in configuration variables is reflected in the corresponding client's `configuration-secure` repository.

**Pull request merged**